flatbush_make_bam_and_qc/split/splitFastq.cwl

```diff
@@ -0,0 +1,60 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: split
+label: split
+
+baseCommand: [ 'python', '/home/reza/flatbush/flatbush_make_bam_and_qc/split/split_lanes.py' ]
```

Comment:
This should be set to a path in the container, not to your `/home/reza` directory